systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java

```diff
@@ -601,5 +619,17 @@ private void saveResultsFile(String filename, Object resultsObject) throws Excep
         Files.write(logsPath.resolve(filename), mapper.writeValueAsBytes(resultsObject));
     }
 
+    private void savePlotCSV(String fileName, Map<String, Object> data) throws IOException {
```

Comment:
this only represents one row CSV, I think to get multiple lines you would need a List<Map<String, Object>> data

Reply:
it must represent one line in csv, plot in jenkins shows continuous progress across builds, so every build is one line